<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageUsersDialog</class>
 <widget class="QDialog" name="ManageUsersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerenciar Usu치rios</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="usersTable"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>Atualizar Usu치rios</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importButton">
       <property name="text">
        <string>Importar Usu치rios</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="syncButton">
       <property name="text">
        <string>Sincronizar Usu치rios</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>